.gitignore

```diff
@@ -73,3 +73,7 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
+
+# access keys
+Client/Keys.swift
```

Comment:
Would be good to add something to the Readme file explaining the use of Keys.swift